pkg/postgres/schema/secured_units.go

```diff
@@ -0,0 +1,41 @@
+// Code generated by pg-bindings generator. DO NOT EDIT.
+
+package schema
+
+import (
+	"reflect"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/postgres"
+	"github.com/stackrox/rox/pkg/postgres/walker"
+)
+
+var (
+	// CreateTableSecuredUnitsStmt holds the create statement for table `secured_units`.
+	CreateTableSecuredUnitsStmt = &postgres.CreateStmts{
+		GormModel: (*SecuredUnits)(nil),
+		Children:  []*postgres.CreateStmts{},
+	}
+
+	// SecuredUnitsSchema is the go schema for table `secured_units`.
+	SecuredUnitsSchema = func() *walker.Schema {
+		schema := GetSchemaForTable("secured_units")
+		if schema != nil {
+			return schema
+		}
+		schema = walker.Walk(reflect.TypeOf((*storage.SecuredUnits)(nil)), "secured_units")
+		RegisterTable(schema, CreateTableSecuredUnitsStmt)
+		return schema
+	}()
+)
+
+const (
+	// SecuredUnitsTableName specifies the name of the table in postgres.
+	SecuredUnitsTableName = "secured_units"
+)
+
+// SecuredUnits holds the Gorm model for Postgres table `secured_units`.
+type SecuredUnits struct {
+	ID         string `gorm:"column:id;type:uuid;primaryKey"`
+	Serialized []byte `gorm:"column:serialized;type:bytea"`
```

Comment:
I'm curious, why aren't we adding the time to the table so we can query instead of walk?

Reply:
I'm not sure the generators will ever cover every permutation of query we may want to run.  Nothing stops us from writing a store that extends the generated one where we could write said query by hand.

Reply:
I was explicitly asked to not write queries by hand and use generated store. I've got a parallel PR 
 #7160 (outdated a bit) with custom store and queries, though.

Reply:
Fair enough.

Reply:
> writing a store that extends the generated one where we could write said query by hand.

If we provide a convenient way to extend a generated store (maybe by offering a `Filter()` function that takes a SQL string that'll be added to the generated store's query `WHERE` clause. I definitely agree that walking all the rows is not a solid long-term solution.

What I've asked not to do is to manually create a store for this use case. The main reason is that we'll have to maintain everything ourselves and also make sure everything works correctly, instead of benefiting from the existing framework for generated stores.